package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "hospitalrun-frontend",
```

Comment:
I think we are going with **@hospitalrun-org** namespace. We can update this to `@hospitalrun-org/frontend`

Reply:
addressed